src/Compilers/CSharp/Test/Emit2/Emit/NumericIntPtrTests.cs

```diff
@@ -11546,6 +11519,27 @@ .maxstack  4
 ");
         }
 
+        [Fact, WorkItem(63860, "https://github.com/dotnet/roslyn/issues/63860")]
+        public void IntPtrOperatorOnNewPlatform()
+        {
+            var source = """
+using System;
+
+class C
+{
+    public static void M()
+    {
+        IntPtr a = default;
+        bool b = a == IntPtr.Zero;
+        a += 1;
+        _ = a + 1;
+    }
+}
+""";
+            var comp = CreateCompilation(source, runtimeFeature: RuntimeFlag.ByRefFields, parseOptions: TestOptions.Regular7_3);
```

Comment:
>CreateCompilation [](http://example.com/codeflow?start=23&length=17)

What ensures the "NewPlatform" usage?

Reply:
Updated the test name. "new platform" indicates a platform where the numeric IntPtr runtime flag is present.